assets/js/views/candidate.js

```diff
@@ -45,8 +50,10 @@ OpenDisclosure.Views.Candidate = Backbone.View.extend({
       if (this.contributions.length > 0) {
         this.renderAllContributions();
       }
+      $('#paymentList').hide();
```

Comment:
This seems pretty weird here - I'd expect it to be in the same place as where that element is shown -- `PaymentCategoryView`. Would that work?